frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 import * as _ from 'lodash';
+import * as Ajv from 'ajv';
+import { JSONSchema6 } from 'json-schema';
```

Comment:
We should have dependencies on these these 2 packages in our `package.json`

Reply:
`json-schema` is being used just for its `JSONSchema6` type definition all over the console. Is it necessary to add it to `package.json` if only type definition is being used?

Reply:
`react-jsonschema-form` has a dependency on `ajv` but it's probably better to make it a direct dependency.

Reply:
Updated `package.json` with new dependencies.